content/pages/en.yaml

```diff
@@ -510,6 +510,518 @@ sections:
+                error_code: |
+                    repeat 100 times print 'How many times can I repeat this?'
+                solution_text: |
+                    Make sure the codes aren't too long
+                solution_code: |
+                    repeat 20 times print 'This is enough'
+    -   level: '8'
+        sections:
+        -   title: "Students use the indentation wrong"
+            example:
+                error_text: |
+                    Indentation is a new concept in this level, that is hard to use for some students. Make sure they practise some simple codes before making a whole program with it.
+                error_code: |
+                    repeat 3 times
+                    print 'hello'
+                solution_text: |
+                    This is the correct code:
+                solution_code: |
+                    repeat 3 times
+                        print 'hello'
+        -   title: "Students only repeat 1 line when they wanted to repeat multiple lines"
+            example:
+                error_text: |
+                    For instance, in the code below the student wanted to take the drinks order of 3 people. But insted the program asked 3 times, but only wrote down one order.
+                error_code: |
+                    repeat 3 times
+                        drink = ask 'What would you like to drink?'
+                    print drink
+                solution_text: |
+                    In the correct code the third line starts with indentation too. This way it belongs to the repeat block and therefore it will be repeated 3 times.
+                    Showing your students these differences can help them understand why we need indentation to make our programs work.
+                solution_code: |
+                    repeat 3 times
+                        drink = ask 'What would you like to drink?'
+                        print drink
+        -   title: "Students want to nest if statements, or put if statements inside a loop"
+            example:
+                error_text: |
+                    In this level students aren't allowed yet to put if statements inside other if statements or inside repeat loops.
+                    In the next level this is allowed.
+                error_code: |
+                    birthday = ask 'Is it you birthday?'
+                    if birthday = yes
+                        repeat 3 times
+                            print 'Hip Hip Hooray!'
+                solution_text: |
+                    This is the correct code for this level:
+                solution_code: |
+                    birthday = ask 'Is it you birthday?'
+                    if birthday = yes
+                        print 'Hip Hip Hooray!'
+                        print 'Hip Hip Hooray!'
+                        print 'Hip Hip Hooray!'
+        -   title: "Students make programs that take too long to run"
```

Comment:
This same mistake is also present for level 7 (albeit that `repeat` and `print` are on the same line in level 7). Is it by design that this mistake is listed for level 7 and 8?